frontend/animalv2/form.js

```diff
@@ -0,0 +1,106 @@
+import _ from "lodash";
+
+import $ from "$";
+
+// general approach lifted from https://stackoverflow.com/questions/501719/dynamically-adding-a-form-to-a-django-formset
+// td#id is duping the cloned row, need to udpate that...
+const cloneSubformRow = function(selector, formIdentifier) {
```

Comment:
We have bits and parts of this all over the HAWC code, but this is nice, and it's nice to have in one place.